client/network/src/service/tests/mod.rs

```diff
@@ -324,6 +334,16 @@ impl TestNetworkBuilder {
 		async_std::task::spawn(async move {
 			let _ = chain_sync_network_provider.run(service).await;
 		});
+		async_std::task::spawn(async move {
+			loop {
+				futures::future::poll_fn(|cx| {
+					import_queue.poll_actions(cx, &mut *link);
+					std::task::Poll::Ready(())
+				})
+				.await;
+				async_std::task::sleep(std::time::Duration::from_millis(250)).await;
```

Comment:
What is the purpose of sleeping for 250 ms here?

Reply:
Just a hack for the tests to let other tasks run as well. I'll see if I can find something better instead

Reply:
The tests use dynamic dispatch for setting up the import queue so there wasn't an easy way to get it working with using `poll_actions` + sleep.